Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -257,31 +239,20 @@ class DatabaseContainer: NSPersistentContainer {
         }
```

Comment:
There's a fatal error here, probably good if we get rid of it.

Reply:
Done ✅